dtfs-central-api/src/v1/controllers/tfm/deal/tfm-update-deal.controller.js

```diff
@@ -139,6 +140,11 @@ exports.updateDealSnapshotPut = async (req, res) => {
 
   const snapshotUpdate = req.body;
 
+  if (snapshotUpdate.dealType === CONSTANTS.DEALS.DEAL_TYPE.BSS_EWCS) {
+    const dealFacilities = await findAllFacilitiesByDealId(dealId);
+    snapshotUpdate.facilities = dealFacilities;
+  }
+
```

Comment:
Do we have a unit test for this?

Reply:
GEF is not required as the format already exists for it. BSS has loans and bonds so we have to treat it differently and inject the facilities into the dealsSnapshot. 
We already had a test for this and I've updated that one to include the dealType BSS/EWCS